### PR TITLE
[Modular] The Syndicate facelift update

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request
A bigass facelift for Syndicate clothing, updating my old stuff, changing some stuff up, removing some other things.

Removal of the secondary Winter coat instead keeping the shorter variant. Updating the MAA gear, ~~giving him his own special pair of Krav Gloves like the station Warden~~ They're in code but not obtainable, along with his own fancy Syndicate police hat.

Image below of the changes.

PR shit

## How This Contributes To The Skyrat Roleplay Experience
Updated sprites cause no one else wants to work on it.

## Changelog

:cl:
imageadd: Resprited Syndicate clothing, check out your new outfit, master at arms
imagedel: Removed one of the 2 Syndicate wintercoats
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm [hunk 1]
```diff
@@ -26,6 +26,13 @@
 	. = ..()
 	AddComponent(/datum/component/toggle_icon)
 
+/obj/item/clothing/head/warden/syndicate
+	name = "master at arms' police hat"
+	desc = "A fashionable police cap emblazoned with a golden badge, issued to the Master at arms. Protects the head from impacts."
```
Comment (ghost) [inline modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm]:
```suggestion
	desc = "A fashionable police cap emblazoned with a golden badge, issued to the Master at Arms. Protects the head from impacts."
```
Shouldn't MaA be capitalized in-sentence like this? (If not feel free to mark this Resolved)

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm [hunk 2]
```diff
@@ -69,6 +76,11 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/hands.dmi'
 	icon_state = "combat"
 
+/obj/item/clothing/gloves/krav_maga/combatglovesplus/maa
+	name = "master at arms' combat gloves"
+	desc = "A set of combat gloves plus emblazoned with red knuckles, showing dedication to the trade, and to hide any bloody knuckles from beating unruly folk."
```
Comment (ghost) [inline modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm]:
```suggestion
	desc = "A set of combat gloves plus emblazoned with red knuckles, showing dedication to the trade while also hiding any blood left after use."
```
Splitting the statement as `A, showing B, and also C` can make the reader believe that either C is unrelated to A, or that B is just a note between A and C. The recommendation (Or similar if you reword it) reads as a simple `A, showing B and C`

File: modular_skyrat/modules/customization/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -48,6 +48,7 @@
 	name = "master at arms's vest"
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/suits/armor.dm]:
```suggestion
	name = "master at arms' vest"
```

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm [hunk 4]
```diff
@@ -26,6 +26,13 @@
 	. = ..()
 	AddComponent(/datum/component/toggle_icon)
 
+/obj/item/clothing/head/warden/syndicate
+	name = "master at arms' police hat"
+	desc = "A fashionable police cap emblazoned with a golden badge, issued to the Master at arms. Protects the head from impacts."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "policehelm_syndie"
```
Comment (ghost) [inline modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm:34]: (Stats from the basetype for reference)
```
armor = list(MELEE = 40, BULLET = 30, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, FIRE = 30, ACID = 60, WOUND = 6)
strip_delay = 60
dog_fashion = /datum/dog_fashion/head/warden
```
	
Would we want to Null the dog_fashion here, or do you think it won't ever come up? Base one will make it a blue police cap with Officer in the name

**Review by ghost — COMMENTED** [review]
**Sprite Review in this comment:**
*I'm not spritetainer, nor a trained spriter. You are a trained spriter. Regardless, you are not exempt from my torturous sprite reviews :trollface:*
Few things that I just wanna double-check are set to go;
- The Combat Gloves/Boots updates are just upstream's current ones, right?
- `modular_skyrat/master_files/icons/mob/clothing/species/teshari/suit.dmi`'s `coatsyndie(_t)` doesn't match `modular_skyrat/master_files/icons/mob/clothing/suit.dmi`'s colors any more. Not planning on updating it in this PR?
- Not sure how to feel about the red stripe on the side-states of `winterhood_syndie`'s mob icons - they look like a harsh outline rather than like detail on the sprite. How do they look ingame?

---
Other than what's been noted, it all looks sound. God bless.

File: modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm [hunk 5]
```diff
@@ -69,6 +76,11 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/hands.dmi'
 	icon_state = "combat"
 
+/obj/item/clothing/gloves/krav_maga/combatglovesplus/maa
```
Comment (tf-4) [inline modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm:80]: Not a fan, we shouldn't be developing ghost roles by station standards. Seems unnecessary.

**Comment by Ebin-Halcyon** [comment]
> **Sprite Review in this comment:** _I'm not spritetainer, nor a trained spriter. You are a trained spriter. Regardless, you are not exempt from my torturous sprite reviews :trollface:_ Few things that I just wanna double-check are set to go;
> 
> * The Combat Gloves/Boots updates are just upstream's current ones, right?
> * `modular_skyrat/master_files/icons/mob/clothing/species/teshari/suit.dmi`'s `coatsyndie(_t)` doesn't match `modular_skyrat/master_files/icons/mob/clothing/suit.dmi`'s colors any more. Not planning on updating it in this PR?
> * Not sure how to feel about the red stripe on the side-states of `winterhood_syndie`'s mob icons - they look like a harsh outline rather than like detail on the sprite. How do they look ingame?
> 
> Other than what's been noted, it all looks sound. God bless.

1. TG doesn't have custom sprites for combat boots/gloves, while Skyrat does.
2. I'm planning to fix most of my updated sprites and updated sprites in general for one big Teshari clothing re-resprite so, not right now.
3. Yeah, swapped that out with the latest commit.
